src/widget/wlibrarysidebar.cpp

```diff
@@ -196,7 +195,7 @@ void WLibrarySidebar::keyPressEvent(QKeyEvent* event) {
 }
 
 void WLibrarySidebar::selectIndex(const QModelIndex& index) {
-    QItemSelectionModel* pModel = new QItemSelectionModel(model());
+    auto  pModel = new QItemSelectionModel(model());
```

Comment:
It looks like the "auto" replacement expects the a 
`QItemSelectionModel *pModel` syntax where it is correct to add a space after *